extensions/objects/templates/boolean-editor.directive.ts

```diff
@@ -44,7 +43,6 @@ angular.module('oppia').directive('booleanEditor', [
       bindToController: {
         value: '='
       },
-      templateUrl: UrlInterpolationService.getExtensionResourceUrl(
-        '/objects/templates/boolean-editor.directive.html'),
+      template: require('./objects/templates/boolean-editor.directive.html'),
```

Comment:
This is wrong, the file is located at `./boolean-editor.directive.html`, ditto for the other files.

Reply:
Done